homeassistant/components/binary_sensor/__init__.py

```diff
@@ -28,6 +28,7 @@
     'gas',           # On means gas detected, Off means no gas (clear)
     'heat',          # On means hot, Off means normal
     'light',         # On means light detected, Off means no light
+    'lock',          # On means locked, Off means unlocked
```

Comment:
Please invert this.

Reply:
?

Reply:
The comment should be inverted:
```py
# On means unlocked, Off means locked
```

Reply:
I think that make no sense on a lock class but for unlock class it make

Reply:
@fabaff  @balloob @andrey-git ?

Reply:
Off is usually the normal state, however with lock I'm not sure which state is normal. I'm fine with this either way.

Reply:
For me a lock is off when it's open and on when locked. I'm more referring to a physical lock or locking device here.
Like with relays they can be both ways: normally open (NO) and normally closed (NC). It all depends on the point of view :smile:

Reply:
Agree with @fabaff and @MartinHjelmare. Typically, we set the states that require attention as `on`, e.g., plug. Accordingly, locked should be `on`.

Reply:
In a security system it's the unlocked locks that needs attention, so unlocked should be `on`.

Reply:
The system is safe when all doors and windows are closed and all locks locked.

Reply:
Agreed with @MartinHjelmare, the majority of device classes are `off` when their state can be disregarded, the `on` state should refer to the state that requires attention, in this case that would mean `unlocked` = `on`, this would follow the same logic as the existing `safety` device class.

Reply:
Okay, I'm with @MartinHjelmare :+1: